chucks-pick3-web/angular-app/src/app/components/customer-summary.component/customer-summary.component.html

```diff
@@ -53,8 +53,8 @@ <h1><b>Customer Summary</b></h1>
         </label>
       </div>
     </th>
-    <th scope="col" class="th-width"> <a [routerLink]="['/customer-details']" routerLinkActive="active" [queryParams]="{showSubNavBar: false}">{{customer.firstName}}</a></th>
-    <th scope="col" class="th-width"><a [routerLink]="['/customer-details']" routerLinkActive="active" [queryParams]="{showSubNavBar: false}">{{customer.lastName}}</a></th>
+    <th scope="col" class="th-width"> <a [routerLink]="['/customer', customer.id]" routerLinkActive="active" [queryParams]="{showSubNavBar: false}" (click)="selectCustomerData(customer)">{{customer.firstName}}</a></th>
```

Comment:
Come up with a better more intuitive method name than `selectCustomerData(customer)` method.  Ask your self what is the user trying to do when they click on this link?  The answer to this question should be reflected in the new method name you come up with .

Reply:
I originally used it to display the customer that is selected via the routerlink and the component but I'm working on a method to simply pull from the main customer array itself.

Reply:
Resolved requested changes.